src/app/item-popup/ItemSocketsGeneral.tsx

```diff
@@ -72,9 +72,14 @@ function ItemSocketsGeneral({
     return null;
   }
 
-  const exoticArmorPerk = item.sockets.categories.find(
-    (c) => c.category.hash === SocketCategoryHashes.ArmorPerks_LargePerk
-  )?.sockets[0];
+  const exoticArmorPerk =
+    item.sockets.categories.find(
+      (c) => c.category.hash === SocketCategoryHashes.ArmorPerks_LargePerk
+    )?.sockets[0] ||
+    (item.bucket.inArmor &&
+      item.sockets.allSockets.find((socket) =>
+        socket.plugged?.plugDef.plug.plugCategoryIdentifier.match(/enhancements\.exotic/i)
```

Comment:
not sure if I should use .test(/enhancements\.exotic/i) or .includes("enhancements.exotic") instead.

Reply:
i lean strongly toward includes and trusting the capitalization not to change